engine/database.py

```diff
@@ -16,21 +16,21 @@ def __init__(self, robot):
             heading_pid: heading PID in format of [proportional factor, integral factor, derivative factor]           
             move_dist: the distance in meters that the robot moves per time dt
             turn_angle: the angle in radians that the robot turns per time dt regardless of time step 
-            plastic_weight: the plastic_weight of the trash the robot has collected
+            plastic_level: the plastic_level of the trash the robot has collected
```

Comment:
I think %'s based on the break beam sensors. Since we have 16 w 4 for each level ill add 25, 50, 75, 100 for now and a TODO to check in the future when electrical has a better idea of how its going to work.